hat/assets/js/apps/Iaso/domains/orgUnits/hooks/requests/useGetGroups.ts

```diff
@@ -12,22 +11,31 @@ import { staleTime } from '../../config';
 type Props = {
     dataSourceId?: number;
     sourceVersionId?: number;
+    blockOfCountries?: string;
 };
 
-const makeGroupsQueryParams = ({ dataSourceId, sourceVersionId }) => {
+const makeGroupsQueryParams = ({
+    dataSourceId,
+    sourceVersionId,
+    blockOfCountries,
+}) => {
     if (sourceVersionId) return `?version=${sourceVersionId}`;
     if (dataSourceId) return `?dataSource=${dataSourceId}`;
+    if (blockOfCountries) return `?blockOfCountries=${blockOfCountries}`;
```

Comment:
you could probably have the type of `blockOfCountries` set to `boolean` and rewrite the condition:
```javascript
if (blockOfCountries) return `?blockOfCountries=True`
```

That way you could pass `true` instead of `True`when you call your hook in the component. It's not really blocking, though